services/movie_session.py

```diff
@@ -42,3 +44,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The 'delete_movie_session_by_id' function uses 'MovieSession.objects.get', which will raise a 'DoesNotExist' exception if the session ID is not found. Consider handling this exception to improve robustness.